docs/v0.9/en/reference/sql/explain.md

```diff
@@ -35,21 +35,51 @@ The column `plan_type` indicates whether it's a`logical_plan` or `physical_plan`
-          CoalesceBatchesExec: target_batch_size=4096, metrics=[output_rows=3, elapsed_compute=43.004µs, spill_count=0, spilled_bytes=0, mem_used=0]
-            RepartitionExec: partitioning=RoundRobinBatch(10), metrics=[fetch_time=5.832958ms, repart_time=10ns, send_time=2.467µs]
-              RepartitionExec: partitioning=RoundRobinBatch(10), metrics=[fetch_time=386.585µs, repart_time=1ns, send_time=7.833µs]
-                ExecutionPlan(PlaceHolder), metrics=[]
-                
+*************************** 1. row ***************************
+stage: 0
+ node: 0
+ plan:  MergeScanExec: peers=[5471788335104(1274, 0), 5471788335105(1274, 1), ] metrics=[output_rows: 0, greptime_exec_read_cost: 0, finish_time: 1496211, ready_time: 846828, first_consume_time: 1491941, ]
+
+*************************** 2. row ***************************
+stage: 1
+ node: 0
+ plan:  CoalesceBatchesExec: target_batch_size=8192 metrics=[output_rows: 0, elapsed_compute: 4147, ]
+  FilterExec: host@2 = host1 metrics=[output_rows: 0, elapsed_compute: 32, ]
+    RepartitionExec: partitioning=RoundRobinBatch(32), input_partitions=8 metrics=[repart_time: 8, fetch_time: 230515, send_time: 256, ]
+      SeqScan: region=5471788335105(1274, 1), partition_count=0 (0 memtable ranges, 0 file ranges) metrics=[output_rows: 0, mem_used: 0, ]
+
+*************************** 3. row ***************************
+stage: 1
+ node: 1
+ plan:  CoalesceBatchesExec: target_batch_size=8192 metrics=[output_rows: 0, elapsed_compute: 3660, ]
+  FilterExec: host@2 = host1 metrics=[output_rows: 0, elapsed_compute: 32, ]
+    RepartitionExec: partitioning=RoundRobinBatch(32), input_partitions=8 metrics=[repart_time: 8, fetch_time: 113774, send_time: 256, ]
+      SeqScan: region=5471788335104(1274, 0), partition_count=0 (0 memtable ranges, 0 file ranges) metrics=[output_rows: 0, mem_used: 0, ]
+
+*************************** 4. row ***************************
+stage: NULL
+ node: NULL
+ plan: Total rows: 0
+4 rows in set (0.002 sec)                
 ```
 
-TODO: explains the output of `ANALYZE`.
+In above example, we got 4 rows of output. Except the last row gives an overall summary, each remains row represents a staged node of the execution plan. The `stage` column indicates the stage number, and the `node` column indicates the node number. The `plan` column explains the plan and related metrics in detail.
+
+### Stage and node
+
+This example is based on a distributed partition table with two regions. We mark different "stage" in a distributed execution plan with stage number, start from 0. And in each stage, we may have multiple "node" to execute the plan in parallel. The node number also starts from 0 in each stage.
+
+### Metrics
+
+Each plan node has its own metrics after the word `metrics`. The metrics may or may not include `output_rows`, `elapsed_compute`, `repart_time`, `fetch_time`, `send_time`, `mem_used` etc. They can be classified into several categories:
```

Comment:
could you explain all these fields' meaning?

Reply:
I don't think this is necessary or viable. There are lots of different metrics and most of them have a not bad name describes what it does

Reply:
To be honest, I don't know. You don't need to list all the fields, but the primary fields deserve to be explained.

Reply:
The most important one is `elapsed_compute`. Do you want something like this?
```
elapsed_compute: time elapsed in computing
```

Reply:
Yes, something like that, including `fetch_time`, `send_time` etc.